monero-outreach-docs/translations/ro/quickfacts/QuickFacts_ro.md

```diff
@@ -0,0 +1,129 @@
+**Nuclee CPU Securizand Reteaua:**14,499,178
+**Monero in Circulatie:**16,570,126 XMR ((Aproximativ)
+**Capitalizare de Piata:**$1,503,928,296 USD (~0.77% din totalul capitalizarii de piata)
+**Recompensa Curenta per Bloc:**3.5984 XMR
+**Durata Medie a Blocului:**2 Minute
+
+_Rata recompenselor va scadea constant pana la sfarsitul lunii mai 2022, cand vor exista 18,132 milioane XMR in circulatie, moment in care o recompensa de 0,6 XMR per bloc va ramane pe termen nelimitat._
+
+_Cu "emisia reziduala" de 0,6 XMR/bloc, pana in 2040 va exista un volum de Bitcoin si Monero egal (aproximativ 21 milioane)._
+
+### CARACTERISTICI IN DEZVOLTARARE
+
+Desi monero este deja disponibil si folosit pe tot globul, comunitatea de dezvoltatori au ca si obiectiv imbunatatirea caracteristicilor de confidentialitate, securitate si utilitate a Monero si criptomonedelor in general. Acestea sunt cateva care vin in curand:
+
+**Bulletproofs:** Aceasta **[dezvoltare](https://getmonero.org/2017/12/07/Monero-Compatible-Bulletproofs.html)** stocheaza tranzactiile mai eficient si permite tranzactii mai rapide.
+
+**Kovri:** Aceasta este o imbunatatire importanta a confidentialitatii, adaugand un al patrulea nivel de confidentialitate tranzactiilor Monero. Kovri utilizeaza **[criptarea tip usturoi](https://getmonero.org/resources/moneropedia/garlic-encryption.html)** (ganditi-va la Tor pe steroizi) pentru a masca adresele IP ale expeditorilor si receptorilor care folosesc reteaua. Fiecare strat al unei tranzactii Monero, de la expeditor, receptor, suma tranzactiei si (cu Kovri) coloana vertebrala a tranzactiei in sine, toate vor fi confidentiale si sigure.
+
+**Portofel Hardware:** Portofelele Ledger si Trezor lucreaza la adaugarea Monero (Ledger este suportat începând cu CLI v0.12.2 si GUI v0.12.3). Comunitatea Monero finanteaza o echipa care **[construieste un portofel hardware](https://btcmanager.com/moneros-hardware-project-moves-into-the-next-phase/)**, construit pe baza principiilor Monero. Ambele optiuni de portofel hardware se asteapta sa fie operationale pana la sfarsitul anului 2018.
+
+**Kasisto:** Aceasta solutie **[punct-de-vanzare](https://github.com/amiuhle/kasisto)** este conceputa pentru a facilita vanzarile pentru comercianti si companii.
+
+
+## RESURSE ADITIONALE
+
+**_[getmonero.org (Official Website)](https://getmonero.org/)_**
+**_[monero.how](https://www.monero.how/)_**
+**_[reddit.com/r/monero](https://www.reddit.com/r/Monero/)_**
+**_[Guide to Monero (post)](https://www.reddit.com/r/CryptoCurrency/comments/7ra409/your_guide_to_monero_and_why_it_has_great/)_**
+**_[Monero In-Depth Technical Intro](https://steemit.com/monero/@sgp/7yjqso-a-monero-introduction-for-beginners)_**
+**_[Zero to Monero](https://www.getmonero.org/library/Zero-to-Monero-1-0-0.pdf)_**
+
+**_[Scams to Avoid](https://www.reddit.com/r/Monero/wiki/avoid)_**
+**_[Monero FAQ](https://ww.getmonero.org/get-started/faq/)_**
+**_[Monero SWOT Analysis]()_**
```

Comment:
Why is this link empty? @lh1008 ?

Reply:
It's empty in all the other files too, and PDFs.

Reply:
Ok, got it.

Reply:
We had a SWOT analysis we found in the web, but suddenly it disappeared from the web. So we deleted the link and are currently working on the SWOT analysis directly from the Monero Outreach workgroup through taiga [#66 SWOT Analysis](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/us/66). :)